CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+# version 3.4.3 GA
+* Update lots of dependencies.
```

Comment:
I think it's ok not to mention here, I'll update the related issues after the release
```suggestion
```